keel-sql/src/main/resources/db/changelog/20201112-add-verification-repository.yml

```diff
@@ -0,0 +1,55 @@
+databaseChangeLog:
+- changeSet:
+    id: add-verification-repository
+    author: fletch
+    changes:
+    - createTable:
+        tableName: verification_state
+        columns:
+          - column:
+              name: environment_uid
+              type: char(26)
+              constraints:
+                nullable: false
+          - column:
+              name: artifact_uid
```

Comment:
Good opportunity for our first foreign key...

Reply:
Could have at least 2 on this table, yes. I'll probably add it in a separate PR.